package-lock.json

```diff
@@ -8837,9 +8837,9 @@
       }
     },
     "postman-collection": {
-      "version": "4.2.1",
-      "resolved": "https://registry.npmjs.org/postman-collection/-/postman-collection-4.2.1.tgz",
-      "integrity": "sha512-DFLt3/yu8+ldtOTIzmBUctoupKJBOVK4NZO0t68K2lIir9smQg7OdQTBjOXYy+PDh7u0pSDvD66tm93eBHEPHA==",
+      "version": "4.4.0",
+      "resolved": "https://registry.npmjs.org/postman-collection/-/postman-collection-4.4.0.tgz",
+      "integrity": "sha512-2BGDFcUwlK08CqZFUlIC8kwRJueVzPjZnnokWPtJCd9f2J06HBQpGL7t2P1Ud1NEsK9NHq9wdipUhWLOPj5s/Q==",
```

Comment:
Why do we have 2 copies of `postman-collection` here?

Reply:
I'm not sure actually. I even did a fresh install (remove package-lock, node_modules and reset cache) on node v16 and it's still being added. I'll try to find more on this but since it's non-blocking should we go ahead with merging PR?